src/molecules/formfields/CreditCardField/index.js

```diff
@@ -48,13 +52,11 @@ function CreditCardField(props) {
           <div className={styles['col']}>{ securityCode }</div>
         </div>
       </div>
-      { meta && meta.touched && meta.error &&
-        <div className={styles['error']}>{ meta.error }</div>
-      }
 
-      {meta && meta.touched && meta.warning &&
-        <div className={styles.error}>{ meta.warning }</div>
-      }
+      <ErrorMessage
+        condition={!!message}
```

Comment:
I meant that the existence of the error message is the conditional. Passing it twice seems redundant to me.